guides/20240819_ai_transcription.md

```diff
@@ -0,0 +1,312 @@
+---
+title: "Transcribe Video and Audio with Python"
+description: "This guide provides a comprehensive step-by-step process for setting up and using Sapat to automatically transcribe video and audio files using Azure OpenAI's Whisper API."
```

Comment:
Maybe to be more generic and just state OpenaAI Whisper compatible API or whatever is correct?